Tests/NIOPosixTests/NonBlockingFileIOTest.swift

```diff
@@ -282,6 +282,14 @@ class NonBlockingFileIOTest: XCTestCase {
         XCTAssertEqual(2, numCalls)
     }
 
+    func testReadMoreThanIntMaxBytesDoesntThrow() throws {
+        // here we try to read way more data back from the file than it contains but it serves the purpose
+        // even on a small file the OS will return EINVAL if you try to read > INT_MAX bytes
+        try withTemporaryFile(content: "some-dummy-content", { (filehandle, path) -> Void in
```

Comment:
Can we expand this test to confirm the content is appropriate? (In this case, empty.)

Reply:
👍I have added a content check here